storops/unity/resource/host.py

```diff
@@ -120,7 +196,52 @@ def get_resource_class(cls):
 
 
 class UnityHostInitiator(UnityResource):
-    pass
+    @classmethod
+    def create(cls, cli, name, host, type, is_ignored=None,
+               chap_user=None, chap_secret=None, chap_secret_type=None):
+
+        if type == HostInitiatorTypeEnum.ISCSI:
+            resp = cli.post(cls().resource_class,
+                            host=host,
+                            initiatorType=type,
+                            initiatorWWNorIqn=name,
+                            chapUser=chap_user,
+                            chapSecret=chap_secret,
+                            chapSecretType=chap_secret_type,
+                            isIgnored=is_ignored)
+        elif type == HostInitiatorTypeEnum.FC:
+            resp = cli.post(cls().resource_class,
+                            host=host,
+                            initiatorType=type,
+                            initiatorWWNorIqn=name,
+                            isIgnored=is_ignored)
+        else:
+            raise UnityHostInitiatorUnknownType(
+                '{} parameter is unknown type'.format(type))
+
+        resp.raise_if_err()
+        return cls(_id=resp.resource_id, cli=cli)
+
+    def modify(self, host, is_ignored=None, chap_user=None,
+               chap_secret=None, chap_secret_type=None):
+        req_body = {'host': host}
+
+        if is_ignored is not None:
```

Comment:
No need to check `None` here.  The parameters with `None` value will be filtered by the client class before we sending the request.